wikipediapreview-wordpress.php

```diff
@@ -4,3 +4,19 @@
  * Description: Add Wikipedia Preview to your articles
  * Version: 0.1.0
  */
+
+function wikipediapreview_enqueue_scripts() {
+    wp_enqueue_script(
+        'wikipedia-preview', 
+        plugin_dir_url( __FILE__ ) . 'node_modules/wikipedia-preview/dist/wikipedia-preview.development.js', [], false, true 
```

Comment:
We obviously can't distribute the plugin with a path like this. How do we make a nice dir structure but also easy for us working on this? Can we try a npm postinstall step that copies what we need from node_modules into `./assets/js` or similar?

Reply:
done here https://github.com/wikimedia/wikipediapreview-wordpress/pull/2/commits/490c7ecb1591b6c29c48762c9aaa57d72803f434
please check if this looks good